Makefile

```diff
@@ -1,3 +1,6 @@
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+# Copyright (C) 2019-2024 Intel Corporation
```

Comment:
Let's make it either small or large but nothing in between. 😉 

```suggestion
# Copyright (c) 2019-2024 Intel Corporation
```

Reply:
Done